client/src/modules/stock/exit/exit.js

```diff
@@ -22,17 +22,8 @@ function StockExitController(
   StockModal, uiGridGroupingConstants, $translate, AppCache
 ) {
   var vm = this;
-  var mapExit = {
-    patient : { description : 'STOCK.EXIT_PATIENT', find : findPatient, submit : submitPatient },
-    service : { description : 'STOCK.EXIT_SERVICE', find : findService, submit : submitService },
-    depot : { description : 'STOCK.EXIT_DEPOT', find : findDepot, submit : submitDepot },
-    loss : { description : 'STOCK.EXIT_LOSS', find : configureLoss, submit : submitLoss },
-  };
-
-  var cache = new AppCache('StockExit');
 
-  vm.util = util;
-  vm.Stock = new StockForm('StockExit');
+  vm.stockForm = new StockForm('StockExit');
```

Comment:
It looks like the `StockForm` is [already defined in the scope](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/client/src/modules/stock/exit/exit.html#L39).  This means that `$scope.StockForm` and `vm.stockForm` are both defined and different things.  To avoid confusion, could you rename this (or the view) as `StockExitForm`?